test/MixedLanguage/tensor.cpp

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+// REQUIRES: nvcc
+
+// RUN: (nvcc -c -Xcompiler -fPIC -std=c++17 %p/tensor.cu -o %t.o && \
+// RUN: nvq++ -std=c++17 --enable-mlir %s %t.o -L `dirname $(which nvcc)`/../lib64 -lcudart -o %t.x && echo "Success") | \
+// RUN: FileCheck %s
```

Comment:
I think you can simplify this.
```
// RUN: <command1>
// RUN: <command2>
```
is just like `<comand1> && <command2>` but a bit easier to read.

And then the `echo | FileCheck` isn't needed. If the compiler fails it will return a non-zero exit code. (If it doesn't that's a bug; but we shouldn't have to assume our stuff is buggy.)

Reply:
done, thanks!